tests/code/test_participants.py

```diff
@@ -301,13 +301,23 @@ def test_getEventParticipantsWithWrongParticipant():
 @pytest.mark.integration
 def test_getUserParticipatedEvents():
     with mainDB.atomic() as transaction:
-        allProgramTrainings = Event.select().join(ProgramEvent).where(Event.isTraining == 1, ProgramEvent.program == Program.get_by_id(2))
+        currentTerm = Term.get_by_id(3)
+        ayStart = currentTerm.academicYearStartingTerm
+
+        allProgramTrainings = (Event.select()
+                                   .join(ProgramEvent)
+                                   .where(Event.isTraining == True,
+                                          ProgramEvent.program == Program.get_by_id(2),
+                                          (Event.term == ayStart) | (Event.term == currentTerm))
```

Comment:
Same comment here as before. Also, we don't want to just repeat the query. We probably should set up data that we know will work (or not work) regardless of what is in test data

Reply:
Resolved